### PR TITLE
User orders

### PR DESCRIPTION
**Pull request by sidewinder2020**
LIST OF CHANGES:
- user story 13,14 and tests
-
-
-

THINGS I'M UNSURE ON:
-
-
-

RSPEC GREEN?
YES
NO- WHY?
YES

*MENTION GROUP MEMBER TO REVIEW PR*

### CODE REVIEW

File: app/models/cart.rb [hunk 1]
```diff
@@ -44,4 +44,15 @@ def decrease_item(item_id)
   def remove_item(item_id)
     @contents.delete(item_id.to_s)
```
Comment (snayrouz) [inline app/models/cart.rb:45]: i like it

File: app/models/cart.rb [hunk 2]
```diff
@@ -44,4 +44,15 @@ def decrease_item(item_id)
...
   end
+
+  def parse_for_orders
```
Comment (snayrouz) [inline app/models/cart.rb:48]: looks like the method for a hash to display the order?

File: app/views/orders/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<h1> past orders </h1>
+
+<% @orders.each do |order| %>
+  <%= link_to "order #{order.id}", order_path(order) %>
+  <%= "total cost: #{order.total_price}" %>
+  <%= "Purchased: #{order.created_at}" %>
```
Comment (blsrofe) [inline app/views/orders/index.html.erb:6]: Nice!

File: spec/features/user_can_view_an_individual_order_spec.rb [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+      expect(current_path).to eq("/orders/1")
+      expect(page).to have_content("order 1")
+      expect(page).to_not have_content("order 2")
+      expect(page).to have_content("Mask")
+      expect(page).to have_content("154.85")
+      # expect(page).to have_content("3")
+      expect(page).to have_css("img[src*='http://www.scuba-info.com/images/scuba_gear_fins.jpg']")
+      expect(page).to have_content("69.95")
+    end
+  end
```
Comment (blsrofe) [inline spec/features/user_can_view_an_individual_order_spec.rb:48]: Awesome test!

**Pull request merged**